leafmap/common.py

```
@@ -16104,27 +16104,26 @@ def h5_to_gdf(
-
-        for key, value in data.items():
-            if columns is None or key in columns or key == lat or key == lon:
-                col_names.append(key)
-                col_val.append(value[:].tolist())
-
-        df = pd.DataFrame(map(list, zip(*col_val)), columns=col_names)
-        out_df = pd.concat([out_df, df])
+        col_data = {
+            key: value[:]
+            for key, value in data.items()
+            if columns is None or key in columns or key == lat or key == lon
```

Ruff (0.16.0)

[warning] Use `key in (lat, lon)` instead of multiple equality comparisons

Replace with `key in (lat, lon)`

(SIM109)